### PR TITLE
chore(devenv): introduce addon-docs

### PR DESCRIPTION
**Pull request by asudoh**
This change introduces Storybook docs add-on, which is a successor of info/notes add-ons. Using docs add-on allows us to reflect `README.md` contents to Storybook.

### CODE REVIEW

File: src/components/tooltip/tooltip-story-react.tsx [hunk 1]
```diff
@@ -37,25 +37,36 @@ const createProps = () => ({
...
+            </BXTooltipBody>
+          </BXTooltip>
+        </>
+      );
+    },
+    {
+      docs: {
+        storyDescription: `
+Interactive tooltip should be used if there are actions a user can take in the tooltip (e.g. a link or a button).
+For more regular use case, e.g. giving the user more text information about something, use definition tooltip or icon tooltip.
```
Comment (emyarod) [inline src/components/tooltip/tooltip-story-react.tsx]:
```suggestion
For more regular use cases, e.g. giving the user more text information about something, use definition tooltip or icon tooltip.
```

File: src/components/tooltip/tooltip-story-vue.ts [hunk 2]
```diff
@@ -44,5 +46,14 @@ storiesOf('Tooltip', module)
...
     `,
-    ...createVueBindingsFromProps(createProps()),
-  }));
+      ...createVueBindingsFromProps(createProps()),
+    }),
+    {
+      docs: {
+        storyDescription: `
+Interactive tooltip should be used if there are actions a user can take in the tooltip (e.g. a link or a button).
+For more regular use case, e.g. giving the user more text information about something, use definition tooltip or icon tooltip.
```
Comment (emyarod) [inline src/components/tooltip/tooltip-story-vue.ts]:
```suggestion
For more regular use cases, e.g. giving the user more text information about something, use definition tooltip or icon tooltip.
```

File: src/components/tooltip/tooltip-story.ts [hunk 3]
```diff
@@ -31,22 +31,33 @@ const createProps = () => ({
...
+            </bx-tooltip-footer>
+          </bx-tooltip-body>
+        </bx-tooltip>
+      `;
+    },
+    {
+      docs: {
+        storyDescription: `
+Interactive tooltip should be used if there are actions a user can take in the tooltip (e.g. a link or a button).
+For more regular use case, e.g. giving the user more text information about something, use definition tooltip or icon tooltip.
```
Comment (emyarod) [inline src/components/tooltip/tooltip-story.ts]:
```suggestion
For more regular use cases, e.g. giving the user more text information about something, use definition tooltip or icon tooltip.
```

File: src/components/tooltip/tooltip-story-angular.ts [hunk 4]
```diff
@@ -44,8 +46,17 @@ storiesOf('Tooltip', module)
...
+      props: createProps(),
+      moduleMetadata: {
+        schemas: [CUSTOM_ELEMENTS_SCHEMA],
+      },
+    }),
+    {
+      docs: {
+        storyDescription: `
+Interactive tooltip should be used if there are actions a user can take in the tooltip (e.g. a link or a button).
+For more regular use case, e.g. giving the user more text information about something, use definition tooltip or icon tooltip.
```
Comment (emyarod) [inline src/components/tooltip/tooltip-story-angular.ts]:
```suggestion
For more regular use cases, e.g. giving the user more text information about something, use definition tooltip or icon tooltip.
```

File: src/components/notification/notification-story.ts [hunk 5]
```diff
@@ -43,64 +43,86 @@ const createToastProps = () => ({
...
+          @bx-notification-beingclosed="${handleBeforeClose}"
+          @bx-notification-closed="${action('bx-notification-closed')}"
+        >
+        </bx-toast-notification>
+      `;
+    },
+    {
+      docs: {
+        storyDescription: `
+Toasts are a non-modal, time-based window elements used to display short messages;
```
Comment (emyarod) [inline src/components/notification/notification-story.ts]:
```suggestion
Toasts are non-modal, time-based window elements used to display short messages;
```

File: src/components/notification/notification-story-vue.ts [hunk 6]
```diff
@@ -91,19 +104,28 @@ storiesOf('Notifications', module)
...
+            },
+            handleClose: action('bx-notification-closed'),
+          };
+        })(createToastProps())
+      ),
+    }),
+    {
+      docs: {
+        storyDescription: `
+Toasts are a non-modal, time-based window elements used to display short messages;
```
Comment (emyarod) [inline src/components/notification/notification-story-vue.ts]:
```suggestion
Toasts are non-modal, time-based window elements used to display short messages;
```

File: src/components/notification/notification-story-react.tsx [hunk 7]
```diff
@@ -45,62 +45,84 @@ const createToastProps = () => ({
...
+          open={open}
+          onBeforeClose={handleBeforeClose}
+          onClose={action('onAfterClose')}
+        />
+      );
+    },
+    {
+      docs: {
+        storyDescription: `
+Toasts are a non-modal, time-based window elements used to display short messages;
```
Comment (emyarod) [inline src/components/notification/notification-story-react.tsx]:
```suggestion
Toasts are non-modal, time-based window elements used to display short messages;
```

File: src/components/notification/notification-story-angular.ts [hunk 8]
```diff
@@ -92,20 +105,29 @@ storiesOf('Notifications', module)
...
+        };
+      })(createToastProps()),
+      moduleMetadata: {
+        schemas: [CUSTOM_ELEMENTS_SCHEMA],
+      },
+    }),
+    {
+      docs: {
+        storyDescription: `
+Toasts are a non-modal, time-based window elements used to display short messages;
```
Comment (emyarod) [inline src/components/notification/notification-story-angular.ts]:
```suggestion
Toasts are non-modal, time-based window elements used to display short messages;
```

File: src/components/button/README.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
+# Button
+
+Buttons are used to initialize an action, either in the background or foreground of an experience.
+
+There are several kinds of buttons:
+
+- Primary buttons should be used for the principle call to action on the page.
```
Comment (emyarod) [inline src/components/button/README.md]:
```suggestion
- Primary buttons should be used for the principal call to action on the page.
```

File: src/components/button/README.md [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+- Secondary buttons should be used for secondary actions on each page.
+- Danger buttons should be used for a negative action (such as Delete) on the page.
+
+Small buttons may be used when there is not enough space for a regular sized button. This issue is most found in tables. Small button should have three words or less.
```
Comment (emyarod) [inline src/components/button/README.md]:
```suggestion
Small buttons may be used when there is not enough space for a regular sized button. This issue is most often found in tables. Small buttons should have three words or fewer.
```

**Comment by asudoh** [comment]
@emyarod Thank you for reviewing!

**Pull request merged**